src/cluster/src/install.rs

```diff
@@ -466,6 +467,44 @@ impl ClusterInstaller {
         }
     }
 
+    /// Create the namespace where fluvio will be installed;
+    /// `fluvio` is the default namespace for fluvio installations;
+    #[instrument(
+        skip(self),
+        fields(namespace = &*self.config.namespace),
+    )]
+    pub async fn create_namespace(&self) -> Result<(), ClusterError> {
```

Comment:
so installer should not create namespace by default.  it should be explicit opt-in by installer as additional argument as convenience.